README.md

```diff
@@ -105,11 +105,16 @@ GSuite provider.
 
 1. Next time you run `terraform plan` it'll use your updated version
 
-Some useful resources:
-
+### Relevant Google Admin SDK Documentation
+#### General
 * http://google.golang.org/api/admin/directory/v1
 * https://developers.google.com/admin-sdk/directory/v1/reference/
 
+#### Schema Types
+* https://developers.google.com/admin-sdk/directory/v1/reference/users
+* https://developers.google.com/admin-sdk/directory/v1/reference/groups
+* https://developers.google.com/admin-sdk/directory/v1/reference/schemas
+
```

Comment:
You may want to add a line here that folks using a service account should add:
`https://www.googleapis.com/auth/admin.directory.userschema` to the `oauth_scopes` list.

Reply:
(I'll fix in merge, don't worry about it)

Reply:
Ah sorry about that. I added it to the default scopes but forgot to add it here too, thanks for spotting and fixing this 👍.